media/css/base/mozid-modal.less

```diff
@@ -20,6 +20,7 @@ html.noscroll body {
     left: 0;
     z-index: 9999999;
     overflow: auto;
+    .animation(sand-fade-in 0.1s ease-in 0s 1 normal both);
```

Comment:
This animation is a little too fast to notice going in, and doesn't provide a nice fade out when dismissing the modal.

I wonder if we could do a class with `opacity: 1` along with a transition, then toggle that class in the JS?